src/audio/codec_adapter/codec/cadence.c

```diff
@@ -172,6 +172,47 @@ int cadence_codec_init(struct comp_dev *dev)
 	return ret;
 }
 
+static int update_stream_params(struct comp_dev *dev)
+{
+	int ret = 0;
+	struct comp_data *ca_data = comp_get_drvdata(dev);
+	struct codec_data *codec = comp_get_codec(dev);
+	struct cadence_codec_data *cd = codec->private;
+	struct sof_ipc_stream_params *stream = &ca_data->stream_params;
+	/* copy stream params localy to ensure container size is correct */
+	uint32_t sample_width = (stream->frame_fmt == SOF_IPC_FRAME_S16_LE) ? 0x10 : 0x20;
```

Comment:
Hexadecimal feels like a weird stylistic choice here (the numbers are simply 16 and 32)